core/src/test/java/org/commonjava/maven/ext/core/fixture/TestUtils.java

```diff
@@ -129,18 +145,21 @@ public static SMContainer createSessionAndManager( Properties p ) throws Manipul
         }
         final MavenSession mavenSession = new MavenSession( container, null, req, new DefaultMavenExecutionResult() );
 
+        mavenSession.getRequest().setPom( pom );
```

Comment:
What happens if pom is null?

Reply:
Setting a `null` pom should be the same as the default (unset) and seems to be OK as long as you don't intend you use it.

The code you wanted `session,getPom().getParentFile()` requires a non-null pom. In that case, for the few tests that didn't use a pom. I just set it to a temp file which makes this call work to get the parent directory. Even though the pom itself is not valid. it's never called.

Reply:
Ok.